mungegithub/mungers/blunderbuss.go

```diff
@@ -125,11 +126,25 @@ func getPotentialOwners(author string, feats *features.Features, files []*github
 			if owner == author {
 				continue
 			}
+			// Inside of a vendor directory, it is possible that
```

Comment:
I did this only as this has the chance of pretty substantially increasing API token turnover if it's not specific to `vendor/` -- if it's not a worry I can make the change global.